### PR TITLE
Make the assertion messages in `test_trial.py` readable

### PR DESCRIPTION
**Pull request by not522**
## Motivation

ref: #5797
To investigate the cause of the fragile tests, this PR makes the assertion messages readable.

## Description of the changes

* Make the assertion messages in `test_trial.py` readable

### CODE REVIEW

**Comment by c-bata** [comment]
@porink0424 Could you review this PR?

**Review by porink0424 — APPROVED** [review]
LGTM!
I confirmed that multiple warning messages are properly displayed in a readable array format.

File: tests/trial_tests/test_trial.py [hunk 1]
```diff
@@ -79,7 +79,8 @@ def test_check_distribution_suggest_uniform(storage_mode: str) -> None:
             trial.suggest_uniform("x", 10, 30)
 
         # we expect exactly one warning (not counting ones caused by deprecation)
-        assert len([r for r in record if r.category is not FutureWarning]) == 1
+        warnings = [r for r in record if r.category is not FutureWarning]
```
Comment (c-bata) [inline tests/trial_tests/test_trial.py:82]: `test_trial.py` imports `warnings` package at L12. How about renaming the variable like below?

```suggestion
        warning_msgs = [r for r in record if r.category is not FutureWarning]
```

**Review by c-bata — CHANGES_REQUESTED** [review]
Thank you for your pull request. I left one suggestion.

File: tests/trial_tests/test_trial.py (hunk 1 above)
Comment (not522) [reply tests/trial_tests/test_trial.py:82]: How about `warning_records`? It is not the message itself.

Comment (c-bata) [reply tests/trial_tests/test_trial.py:82]: Sounds better to me 👍

**Comment by not522** [comment]
I renamed `warnings` to `warning_records`. PTAL.

**Review by c-bata — APPROVED** [review]
LGTM.

**Pull request merged**